app/controllers/articles_controller.rb

```diff
@@ -12,13 +16,17 @@ def new
   end
 
   def create
-    @article = Article.new(article_params)
+    p helpers.current_user.id
```

Comment:
@JuanMaGiraldo are you using the `git add --patch`? ... please remove this debugging line